<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AardView</class>
 <widget class="QMainWindow" name="AardView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aardview</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QScrollArea" name="imageArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>68</width>
           <height>278</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="imageName">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="imageContainer">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>366</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrintPreview"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionEditCurrentImage"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionToggleFitToWindow"/>
    <addaction name="separator"/>
    <addaction name="actionToggleMenuBar"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dockDirectoryTree">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Directory tree</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="ATreeView" name="dirView"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockTreeView">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Thumbnail view</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="AListView" name="tnView"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockTaggedItems">
   <property name="windowTitle">
    <string>Tagged items</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="AListView" name="tagView"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockStatusInfo">
   <property name="windowTitle">
    <string>Status information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="text">
    <string>P&amp;rint preview</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit Aardview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionEditCurrentImage">
   <property name="text">
    <string>&amp;Edit current image</string>
   </property>
   <property name="toolTip">
    <string>Edit current image in external editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>P&amp;references</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>Show the applications About box</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="toolTip">
    <string>Show the Qt library's about box</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionToggleFitToWindow">
   <property name="text">
    <string>Toggle 'fit to window'</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionToggleMenuBar">
   <property name="text">
    <string>Toggle menu bar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Mi&amp;nimize</string>
   </property>
  </action>
  <action name="actionMaximize">
   <property name="text">
    <string>Ma&amp;ximize</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>&amp;Restore</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AListView</class>
   <extends>QListView</extends>
   <header>alistview.h</header>
  </customwidget>
  <customwidget>
   <class>ATreeView</class>
   <extends>QTreeView</extends>
   <header>atreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionEditCurrentImage</sender>
   <signal>triggered()</signal>
   <receiver>AardView</receiver>
   <slot>openEditor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMinimize</sender>
   <signal>triggered()</signal>
   <receiver>AardView</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMaximize</sender>
   <signal>triggered()</signal>
   <receiver>AardView</receiver>
   <slot>showMaximized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRestore</sender>
   <signal>triggered()</signal>
   <receiver>AardView</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleMenuBar</sender>
   <signal>triggered()</signal>
   <receiver>AardView</receiver>
   <slot>toggleMenuBar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>AardView</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrint</sender>
   <signal>triggered()</signal>
   <receiver>AardView</receiver>
   <slot>print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrintPreview</sender>
   <signal>triggered()</signal>
   <receiver>AardView</receiver>
   <slot>printPreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aboutQt()</slot>
  <slot>openEditor()</slot>
  <slot>toggleMenuBar()</slot>
  <slot>open()</slot>
  <slot>print()</slot>
  <slot>printPreview()</slot>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>toggleFtw()</slot>
  <slot>rotate()</slot>
 </slots>
</ui>
